test/e2e/external_gateways.go

```diff
@@ -1561,14 +1563,18 @@ var _ = ginkgo.Describe("External Gateway", func() {
 					time.Sleep(1 * time.Second)
 
 					ginkgo.By("Post-Migration: Sending Egress traffic and verify it is received")
+					// We don't want traffic to hit the already existing conntrack entry (created for source port 50000)
```

Comment:
shouldn't the mac of the existing conntrack entry get updated when the external gateway sends a packet? I would rather do that then just use a new source port/connection.

Reply:
The mac of the existing conntrack entry will get updated when the external gateway sends a packet on that specific connection.  In the test we just send a GARP/NDP.  That won't update the conntrack entry.

Reply:
Right, but we lose coverage of ensuring that a current connection gets updated correctly when an exgw gets migrated. If you can point to coverage elsewhere in the tests then I'm fine with what you did, otherwise I think we need to send a packet from the exgw to update the conntrack connection and ensure it still works.

Reply:
Also, for TCP it would be cumbersome to inject a packet from the external gateway that refreshes the mac stored in the conntrack entry in the current test.  I'll open an issue to add a dedicated e2e test to cover that scenario.  The current test is about GARP/NDP being properly processed by the gateway router (OVN MAC_Binding should be updated).